### DISCUSSION TITLE
Save the currently-following-entity into save files.

### DISCUSSION THREAD

**Discussion by MaxMallon** [OpenRCT2/OpenRCT2#21426] [Feature requests]
By saving the following entity-id into the savefile, you can more easily share parks with a showcase of a ride. Rather than posting separate instructions as to how to best view the park (with a 'ride' on the ride you made), which can get lost in time, the game would simply start following the ride of your choice.

**Comment by Harry-Hopkinson** [comment]
I have implemented this request in pull request - https://github.com/OpenRCT2/OpenRCT2/pull/21656

**Comment by ZehMatt** [comment]
~~Plugins have enough functionality to get this working.~~ I stay corrected, I was quite confident that all the API is already available, however I still think its best if plugins were to implement this very specific feature.
